src/simplewallet/simplewallet.cpp

```diff
@@ -6408,6 +6409,13 @@ bool simple_wallet::lns_buy_mapping(const std::vector<std::string>& args)
     info.address                        = m_wallet->get_subaddress({m_current_subaddress_account, 0});
     info.is_subaddress                  = m_current_subaddress_account != 0;
     dsts.push_back(info);
+
+    std::cout << std::endl << tr("Buying Loki Name Service Record") << std::endl << std::endl;
+    std::cout << boost::format(tr("Name:         %s")) % name << std::endl;
+    std::cout << boost::format(tr("Value:        %s")) % value << boost::format(tr(" for %s")) % "Session" << std::endl;
+    std::cout << boost::format(tr("Owner:        %s")) % (owner.size() ? owner : m_wallet->get_subaddress_as_str({m_current_subaddress_account, 0}) + " (this wallet) ") << std::endl;
+    std::cout << boost::format(tr("Backup Owner: %s")) % backup_owner << std::endl;
```

Comment:
If backup owner is not specified, print `(none)` so we can be consistent with the ui